src/Runner.php

```diff
@@ -169,6 +181,12 @@ public function run($input = null, $output = null, $app = null, $commandFiles =
                 $commandFiles = [];
             }
         }
+
+        /** @var \Robo\ClassDiscovery\RelativeNamespaceDiscovery $discovery */
+        $discovery = Robo::service('relativeNamespaceDiscovery');
+        $discovery->setRelativeNamespace($this->commandFileRelativeNamespace);
+        $commandFiles = array_merge((array)$commandFiles, $discovery->getClasses());
+
```

Comment:
So this is how we integrate command discovery at the moment, we can/should turn this into a separate service.

Reply:
For now just put it in a separate protected method. Most folks won't need to override this behavior, but it would be better to separate this out anyway.